silk-react-components/src/HierarchicalMapping/Components/MappingRule/ObjectMappingRule.jsx

```diff
@@ -284,6 +291,28 @@ const ObjectRule = React.createClass({
         let entityRelation = false;
         let deleteButton = false;
 
+        const copyButton = isCopiableRule(this.props.type) ? (
```

Comment:
this can be replaced with `const copyButton = isCopiableRule(this.props.type) && <Button
                className="ecc-silk-mapping__rulesviewer__actionrow-copy"
                raised
                onClick={this.handleCopy}>
                Copy
            </Button>`